src/sentry/static/sentry/app/views/organizationIntegrations/integrationDirectoryAppInstallationDetail.tsx

```diff
@@ -0,0 +1,37 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import IntegrationDirectoryApplicationRow from 'app/views/settings/organizationDeveloperSettings/sentryApplicationRow/integrationDirectoryApplicationRow';
+import SentryTypes from 'app/sentryTypes';
+import {Organization, SentryApp, SentryAppInstallation} from 'app/types';
+
+type Props = {
+  organization: Organization;
+  install?: SentryAppInstallation;
+  app: SentryApp;
+};
+
+class SentryAppInstallationDetail extends React.Component<Props> {
```

Comment:
@NisanthanNanthakumar You probably can just do: 
```
React.Component< IntegrationDirectoryApplicationRow['props']>
```
Though I'm not 100% that will work

Reply:
@scefali how to add the `isOnIntegrationPage` prop?

Reply:
@scefali I'm going to refactor this class into a functional component. I just realized there's no state handling